src/Shared/CommunicationsUtilities.cs

```diff
@@ -326,17 +326,17 @@ internal static long GenerateHostHandshakeFromBase(long baseHandshake, long clie
         /// Magic number sent by the host to the client during the handshake.
         /// Derived from the binary timestamp to avoid mixing binary versions.
         /// </summary>
-        internal static long GetTaskHostHostHandshake(TaskHostContext hostContext)
+        internal static long GetHostHandshake(TaskHostContext hostContext)
```

Comment:
I think this is a good rename; you can derive from the fact that it's passed a `TaskHostContext` that it applies to `TaskHost` handshaking.